reference/features/search_parameters.md

```diff
@@ -42,6 +42,12 @@ When `q` isn't specified, a **placeholder search** is performed. Placeholder sea
 
 Placeholder search returns the documents that best match other search parameters, sorted according to that index's [ranking rules](/reference/features/settings.md#ranking-rules). This feature also supports [faceting](/reference/features/faceted_search.md) and [filtering](/reference/features/filtering.md)
 
+### Phrase search
+
+You can enclose search terms in double quotes (`"`), in which case MeiliSearch will only return documents containing all terms in the order they were given. This is called a [phrase search](/learn/what_is_meilisearch/features.md#phrase-search).
+
+Phrase searches are case-insensitive and ignore soft separators such as `-`, `,`, and `:`. You can mix them with regular queries: MeiliSearch will first fetch all documents with exact matches and [then proceed with its default behavior](/learn/core_concepts/relevancy.md).
+
```

Comment:
We spoke about soft separators, but hard separators are handled differently. When we have a Hard separator in a phrase search, we split the phrase into 2 different phrases.
`hello "world! Nice to see you"` would be considered as `hello "world" "Nice to see you"`.
Because in meilisearch, a Hard separator is considered as a split of 2 unrelated contexts.

Reply:
Thanks for that, @ManyTheFish — I'll add an extra sentence about hard separators.